City-of-Helsinki/drupal-hdbt
# UHF-2697: Fix hero margin-bottom when there is sidebar

Hero margin-bottom was 0 when there is a sidebar. This PR fixes that problem.

![image](https://user-images.githubusercontent.com/1191667/141258140-d29cc4f4-360b-45aa-bb5a-a046724f6594.png)

## How to test
* Get this branch
* Create a basic page with a sidebar and a hero (this applies to all hero versions the same)
* Check that the hero does not touch the sidebar or adjacent content.
